app/views/datasets/_results.haml

```diff
@@ -17,7 +17,7 @@
         - @datasets.each do |dataset|
           %tr
             %td
-              .title_result= link_to "#{truncate(dataset.title, length: 50)} - #{dataset.author.name}", dataset
+              .title_result= link_to truncate(dataset.title, length: 50) + " - " + dataset.author.name, dataset
```

Comment:
So `length` was the problem?

Reply:
i think using `truncate` in string interpolation was the problem. so length is still there, and shouldn't break anymore.